LoopOnboardingKitUI/View Controllers/OnboardingUICoordinator.swift

```diff
@@ -361,6 +361,11 @@ class OnboardingUICoordinator: UINavigationController, OnboardingNotifying, CGMM
             self?.stepFinished()
         }
     }
+
+    // MARK: - PreferredGlucoseUnitObserver
+    func preferredGlucoseUnitDidChange(to preferredGlucoseUnit: HKUnit) {
```

Comment:
I can't wait until `preferredGlucoseUnit` is a `@StateObject` (or `@ObservableObject`, or whatever we decide is the best `Combine` riff to use...)